### ISSUE TITLE
Streak Reset for no reason

### ISSUE THREAD

**Issue by janaSunrise** [DenverCoder1/github-readme-streak-stats#83]
**Describe the bug**
I'm contributing and coding everyday, but today it reset all of a sudden for no reason.

**Expected behavior**
It was supposed to continue

**Screenshots**
My contribution graph:
![Screenshot from 2021-04-20 20-37-47](https://user-images.githubusercontent.com/56873669/115420137-83423080-a218-11eb-9f9b-3be5b73fe29c.png)

My stats card:
![Screenshot from 2021-04-20 20-39-44](https://user-images.githubusercontent.com/56873669/115420223-98b75a80-a218-11eb-860d-078f7706841b.png)

**Desktop (please complete the following information):**
 - OS: Linux (Ubuntu)
 - Browser: Firefox
 - Version: latest

**Additional context**

**Comment by janaSunrise** [comment]
Got it fixed by re-adding, Thank you, and sorry!!

**Comment by DenverCoder1** [comment]
Hi @janaSunrise

It looks like this may have been caused by a commit that you have on October 16, 2021.

![image](https://user-images.githubusercontent.com/20955511/115432634-4887ca80-a20f-11eb-845b-5c4781e35675.png)

October 16 is the last item in the generated list so it is treated as the current date and it becomes the current streak.

I think it might make more sense to simply ignore commits after the current date so the current streak makes more sense.

**Comment by janaSunrise** [comment]
Oh, weird! I didnt notice that, I'll try to figure that and fix it! Thanks for your immediate help, Appreciate it! :D

**Comment by DenverCoder1** [comment]
@janaSunrise I only now noticed that you mentioned that you wanted to fix it. Sorry if I took away the opportunity by making the change to the code.

With the new code, your card should display like this with the end date being today's date (UTC)

![image](https://user-images.githubusercontent.com/20955511/115446373-d075d080-a21f-11eb-9c56-3803a114154d.png)

It may take some time for the cached version to get refreshed by GitHub.

Let me know if you have any ideas on ways it can be improved further! :smiley:
